### ISSUE TITLE
Won't save more than 7 accounts

### ISSUE THREAD

**Issue by Avery1337** [Haaxor1689/vanilla-autologin#4]
Pretty much what the title says.  It will keep the 8th account but on next exit and login it isn't saved inside of it.

**Comment by Haaxor1689** [comment]
Sadly there is a hard 256 character limit for the config field so you can't save too many logins there

**Comment by Avery1337** [comment]
Could a new line be made to create another login?

**Comment by Haaxor1689** [comment]
nope, it's saving into the config.wtf file because it's using the same field as "remember account" field and that is limited and I don't know of any other API that I could use to save that login info to from login screen

**Comment by Avery1337** [comment]
This is still a really nice addition to my wow experience.

Would it be possible to have an option give each account a nickname and be able to click on the nickname instead of the acc and blurred out password?

**Comment by Haaxor1689** [comment]
hmm that would require even more space per sadly
